articles/service-fabric/service-fabric-reverseproxy.md

```diff
@@ -104,7 +104,7 @@ Generally when a service cannot be reached it means the service instance or repl
 However, replicas or service instances can share a host process and may also share a port when hosted by an http.sys-based web server, including:
 
 * [System.Net.HttpListener](https://msdn.microsoft.com/library/system.net.httplistener%28v=vs.110%29.aspx)
-* [ASP.NET Core WebListener](https://docs.asp.net/latest/fundamentals/servers.html#weblistener)
+* [ASP.NET Core WebListener](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/servers/weblistener)
```

Comment:
Hello @wagsousa -- I was just passing thru and looking at some PR's in the repo and noticed this small nit: You should remove the `en-us` segment from the link to localize it for outside of the US.